libs/hacl/config.cmake

```diff
@@ -195,20 +195,17 @@ set(INCLUDES
 	${PROJECT_SOURCE_DIR}/include/Hacl_EC_K256.h
 	${PROJECT_SOURCE_DIR}/include/Hacl_FFDHE.h
 	${PROJECT_SOURCE_DIR}/include/internal/Hacl_Impl_FFDHE_Constants.h
-	${PROJECT_SOURCE_DIR}/include/Hacl_SHA3.h
-	${PROJECT_SOURCE_DIR}/include/Hacl_Streaming_SHA3.h
+	${PROJECT_SOURCE_DIR}/include/internal/Hacl_Hash_SHA3.h
```

Comment:
i was getting inconsistent build errors with these changes and so i jsut replicated the default files from hacl. #104 is still a thing